### PR TITLE
Synthetic Durability and Strength

### PR DESCRIPTION
**Pull request by QuickLode**
## About The Pull Request
This PR increases the burn resistance of Synthetics and increases their unarmed combat damage. This is to both to curb a certain strategy which was able to near-instantly eliminate Synthetics, and to follow Aliens-media aesthetics. That includes less weapons carried and sturdier construction.

## Why It's Good For The Game

Synthetics look like clowns carrying giant bulky weapons when they're clearly capable of protecting themselves without them in media. Deincentivizing the need to carry weapons means we can be closer to the aesthetic of Synthetics in Aliens media, such as Samuels and Bishop.

Acid attacks were found to be too powerful against synthetics through certain strategies, able to kill in an extraordinarily short amount of time. A T2 caste was able to severely outperform T3s in terms of DPS, able to near-instantly kill Synthetics with certain strategies. Colony Synthetics are expected to enter hazardous environments such as exposed reactors, vented areas, corrosive materials and fire hazards. We see this especially in Alien:Isolation from the famous line of "I am built to withstand temperatures of up to 1210 degrees."

## Changelog

:cl: QuickLoad
balance: synthetic unarmed damage increased to disincentivize carrying weapons.
balance: synthetics are more resistant against burns
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/unarmed_attacks.dm [hunk 1]
```diff
@@ -39,6 +39,10 @@
 	attack_verb = list("punch","bust","skewer")
 	damage = 10
 
+/datum/unarmed_attack/punch/synthetic
+    attack_verb = list("punch","strik","slugg","bludgeon","impact","maul")
```
Comment (carlarctg) [inline code/modules/mob/living/carbon/human/unarmed_attacks.dm:43]: ???

**Comment by morrowwolf** [comment]
I'd like a bit more conversation on this after people cool out in the feedback thread. That's not to say I agree or disagree with anything.

**Review by fira — APPROVED** [review]
no code issues, need balance look maybe, dunno

**Comment by Geevies** [comment]
my personal thoughts on the balance:
 - drop the extra punch damage entirely, i know lorewise synths are fairly robust but i don't think making the support class naturally strong presents the right idea
 - add the increased burn resistance to working joes, not colony synths

**Pull request merged**